### ISSUE TITLE
Donations

### ISSUE THREAD

**Issue by ghost** [FreshRSS/FreshRSS#1694]
Hi, I was wondering why there is no donation option for FreshRSS. I know there are other ways to contribute, but accepting donations shouldn't hurt.

Of course, the feasibility of that would depend on the size of your userbase and the supported payment methods, but you might want to give it a thought.

**Comment by Alkarex** [comment]
Thanks for the suggestion, @notfoss 👍 

Hum, I think @marienfressinaud had something during a period, didn't you Marien?

Do you have any suggestion of system, which would be appropriate?

It would indeed be good, in particular for motivation :-) We could use the funds for the few expenses (e.g. freshrss.org) and also contribute (at least symbolically) to the libraries / projects / organisations, which we love (such as Framasoft).
If we ever receive anything substantial, it could be nice to use it to motivate an open source artist to refresh our theme(s).

**Comment by Alkarex** [comment]
Maybe https://liberapay.com/

**Comment by marienfressinaud** [comment]
Yes, we used flattr.com for some time but I closed donations and withdrawn the 68€ on the account. If we reopen donations, I suggest to use Liberapay indeed: we can create a FreshRSS team and share donations. Team behind Liberapay is really great and are thinking actively how to support free software communities.

**Comment by ghost** [comment]
I hadn't heard of Liberapay earlier, but it seems FOSS friendly. Now, one issue is that they don't seem to have any one-time donation option and secondly, one needs to create yet another online account to use them.

I was thinking more along the likes of Paypal (yeah, I know many in the FOSS community despise them, but they seem to be the easiest and with the maximum reach).

If possible, it would be great if you can have multiple donation processors.

**Comment by Alkarex** [comment]
Good points @notfoss 

@marienfressinaud Could you summarize the drawbacks of Flattr?

**Comment by marienfressinaud** [comment]
My biggest concern was about the fairness of the system: I was not comfortable with getting all the donations while other contributors were doing great work too. I may maintain the servers and domain name, but that's nothing compared to the time spent on code (freshrss.org is 16,78€/year + server 4,79€/month).

Also, I think to remember Flattr takes 10% of fees on each donation, which is quite a lot I think.

I'm not completely against Paypal, I understand benefit of not creating a new account for every project you want to support. However, the problem of fairness stays :(.

Concerning one-time donation, it may be a problem indeed. It's a principle of Liberapay: the idea is to support FOSS projects on the long-time. I think they are working on a workaround by adding the possibility to make donations per year.

I have not perfect solution. Today, my personal preference goes to Liberapay, but the one who is motivated to set up one or several donation processors can decide :).

**Comment by Alkarex** [comment]
In addition to some suggestions above https://github.com/FreshRSS/FreshRSS/issues/1694#issuecomment-345548285 (also related to fairness, by using the funds on some shared interests),  a related matter could be to start considering hosting some user accounts on a slightly larger server - maybe we could discuss that sometime, also in relation with e.g. CHATONS.

**Comment by marienfressinaud** [comment]
I forgot to precise that I opened registrations for few accounts on rss.freshrss.org, it's up to us to make it really official ;). I would just like to finish to migrate the server first

**Comment by Alkarex** [comment]
Excellent initiative @marienfressinaud . Not so important, but before opening too much, you might want to give a try to another distro with an up-to-date PHP7+, as the performance is so much higher than PHP5.

**Comment by marienfressinaud** [comment]
Yes I know and it is definitely in my plans ;). I'll probably bet on a Debian distro with PHP 7 indeed

**Comment by vincent-ferotin** [comment]
+1 for allowing donations, and +1 for allowing different methods
(Hi, I'm a very fresh new user of FreshRSS, and 'd love making a donation ;-)

**Comment by iamarya2k24** [comment]
Would love to donate few bucks! (And more in future thanks for creating such an awesome software!)

**Comment by Alkarex** [comment]
So, I have finally started https://liberapay.com/FreshRSS/
Comments welcome
@marienfressinaud @aledeg @kevinpapst @Frenzie @pattems Has any of you an account there already?

**Comment by Frenzie** [comment]
I don't have an account yet. Was this the monthly subscription type service?

**Comment by Alkarex** [comment]
Icon added in https://github.com/FreshRSS/FreshRSS/pull/2192
@marienfressinaud Maybe you could consider adding one of the widgets https://liberapay.com/FreshRSS/widgets/ on freshrss.org

@Frenzie Yes it is. I would also like to see a one-time donation. However, it seems to be quite well handled, with good options to support several projects, to split donations between different contributors, etc. I also like the fact that they are a non-for-profit organisation.

**Comment by kevinpapst** [comment]
+1 for one time donations - I'll have a look at that site, never heard of it before

**Comment by pattems** [comment]
I haven't heard of librepay before, either. I'll have to check it out. I'll keep an eye out for anything that looks suitable for one-time donations.

**Comment by marienfressinaud** [comment]
Well, you should note that since they had trouble with Mangopay (you can learn more here: https://medium.com/liberapay-blog/liberapay-is-in-trouble-b58b40714d82), the system is quite different. We'll have to create an account on Paypal or Stripe to be able to receive the money, and while Liberapay will continue to send weekly emails to indicate how much money have been donated, the money itself is transferred in just one transaction to our Paypal/Stripe wallet (explanations here:
https://medium.com/liberapay-blog/liberapay-status-update-a76403f4e0)

So it's a bit different than before and it's a mix between one-time donation and weekly subscription… I don't know what's your thinking about that (I'm still fine with this solution)

**Comment by marienfressinaud** [comment]
I don't know if it's a priority, I'm not the one who continue to make FRSS living ^^' But to be transparent on the costs: 16,79 (domain name) + 57,46 (server) = 74,25 € per year / 52 = **~1,43 € per week**

So once there will be enough donations, I'll take only this amount :)

**Comment by kevinpapst** [comment]
I don't know which topic it was, but I suggested that already in the past: you could move the complete hosting to GitHub including SSL. This way all team member could work & publish on the website and you get rid of the costs at the same time - only the DNS would be needed.

**Comment by marienfressinaud** [comment]
It is a possibility for the website indeed, but it can't be for the demo instance and the update server (+ a real instance that I plan to open more widely in the coming months). Also I like the idea to not be too dependent of GitHub :) So considering the subject (donations and the cost of hosting), it would change nothing to host the website on another server ^^

**Comment by Alkarex** [comment]
Various optimisations might indeed be possible if desired/necessary (while staying on non-devil providers 👼 ) - especially as long as the server is not used much. Just as examples: [13,48€/year for a `.org`](https://www.bookmyname.com/offres.cgi?wl=en) and [3,59€/month for a VPS](https://www.kimsufi.com/en/) (both including VAT).
@marienfressinaud Wouldn't you feel like starting a [CHATONS](https://chatons.org/en)? :-)

**Comment by marienfressinaud** [comment]
I didn't plan to change anything, it's not very expensive and I'm fine with the current configuration :)

Concerning the CHATONS-thing… well, I might have some news soon ;)

**Comment by Alkarex** [comment]
> Concerning the CHATONS-thing… well, I might have some news soon ;)

Suspense! 😛
